src/navigation.tpl

```diff
@@ -23,7 +23,14 @@
                 <li><a href="help-rackspace.html">Rackspace</a></li>
             </ul>
         </li>
-        <li><a href="controllers.html">Controllers</a></li>
+        <li class="section><a href="controllers.html">Controllers</a>
+            <ul class="sub">
+                <li><a href="controllers-creating.html">Creating/Bootstrap</a></li>
```

Comment:
We have a gerund/imperative combo here with Creating/Bootstrap. Should we pick one form and use it for both? Either "Create/Bootstrap" or "Creating/Bootstrapping"?